git-github-introduction.md

```diff
@@ -0,0 +1,340 @@
+
+
+
+
+
+## Git Installation
+Github provides platform-specific installers for Mac and Windows users.  Installation instructions
+are provided in the links below for each supported platform.
+
+[Mac](https://mac.github.com)
+
+[Windows](https://windows.github.com)
+
+[Linux](http://git-scm.com/download/linux)
+
+
+Linux users may install Git using their distribution's package manager.
+Installation commands for many Linux distributions are available at [Linux](http://git-scm.com/download/linux).
+
+For example, Debian-based Linux distributions can use [APT](http://en.wikipedia.org/wiki/Advanced_Packaging_Tool) (Advanced Packaging Tool).
+```
+$ sudo apt-get install git
+```
+
+
+## Configure Git User Information
+
+After installing Git, the next step is for the user to configure his/her author information (username and email address) so that Git will correctly log the author of a commit. 
+
+
+To configure Git author information, open the Terminal and type **git config --global user.name "user's name"**, which will allow you to configure Git on your local machine. Each repository you create will use the same user (author) configuration if you use the global flag (as in the provided example). 
+
+```
+$ git config --global user.name "John Smith"
+
+$ git config --global user.email "john.smith@example.com"
+```
+
+You can check your user information by typing the following command:
+
+```
+$ git config --list
+```
+
+
+## Configure Github User Account
+Before a pull request can be submitted to a [SeattleTestbed](https://github.com/SeattleTestbed)
+repository, a user account (free) must be created on Github.  This user account is also needed to create
+new repositories and fork ones that already exist on Github.
+
+
+A Github user account can be created at https://github.com/join
+
+Once users create their accounts, user-specific home pages are created. For example: ''https://github.com/john-smith''.
+A user-specific home page can contain repositories that the user has created or forked, contribution activity,
+and organizations he/she has joined. 
+
+
+## Fork a `SeattleTestbed` Repository
+
+Forking refers to making a copy of a repository and saving it to your user home page. General instructions on forking repositories and managing pull requests are available [here](https://help.github.com/articles/fork-a-repo)
+
+Forking a [SeattleTestbed](https://github.com/SeattleTestbed) repository requires clicking the ''fork'' button on the selected repository's Github page, which creates a copy of the repository and saves it to a repository directly controlled by the user (on the user's Github home page).  The URL of the forked repository can then be used with Git to create and save a local copy.  
+
+Navigate to the [repy_v2](https://github.com/SeattleTestbed/repy_v2) repository and click on the ''fork'' button in the upper right-hand corner of the page.
+
+
+![GitHub Fork](/images/fork_sm.png)
```

Comment:
Should we update this (and also below) screenshots? GitHub UI has slightly changed since the screenshots were taken.